### PR TITLE
Feat - Added posts page

### PR DESCRIPTION
**Pull request by SRUJANA-PENUGONDA13**
Added posts page with below functionalities

-  Added basic navigation to Explore, Bookmark and profile page
- Adding new posts to feed
- Added private routes for explore, bookmark and profile pages

https://user-images.githubusercontent.com/72279128/177043777-c4fbf077-5fb1-4afa-bc05-8aa9441fdbd5.mp4

### CODE REVIEW

File: src/frontend/components/postsContent/PostsContent.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.posts-middle-sec {
+  /* border: 2px solid var(--text--primary--color); */
```
Comment (simran61) [inline src/frontend/components/postsContent/PostsContent.css]: Can remove comments if not required

File: src/frontend/components/postsContent/PostsContent.css [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  border-radius: 10px;
+  gap: 20px;
```
Comment (simran61) [inline src/frontend/components/postsContent/PostsContent.css]: Try using relative units instead of pixels for values more than 10px.

**Review by simran61 — APPROVED** [review]
Looks good.
Functionality working as expected.
Clean folder structure.

File: src/frontend/components/leftNavBar/LeftNavBar.css [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+.posts-brandName {
+  font-size: 2rem;
+  font-weight: 500;
+  padding: 3rem 1rem;
+}
+
+.posts-nav-links {
+  padding-left: 3rem;
+}
```
Comment (irajbhinde) [inline src/frontend/components/leftNavBar/LeftNavBar.css:19]: Can you add multiple posts & check if Main Nav / Left Nav & Right Nav are sticky or not. Basically, check if the required sticky stylings are working or not

**Review by irajbhinde — COMMENTED** [review]
UI is looking really good, keep up the good work Srujana. Few feedback from my side - 

1. Make the Sticky components Sticky
2. Add redirects file
3. I Am not sure if the user can access the posts page without logging in, user should be able to access the posts page without logging in, just when he clicks on posts / like / bookmark etc he should be redirected to auth
4. Add an explore button on landing page where user can explore the content

File: src/frontend/components/postsContent/PostsContent.css (hunk 2 above)
Comment (SRUJANA-PENUGONDA13) [reply src/frontend/components/postsContent/PostsContent.css]: sure

File: src/frontend/components/leftNavBar/LeftNavBar.css (hunk 3 above)
Comment (SRUJANA-PENUGONDA13) [reply src/frontend/components/leftNavBar/LeftNavBar.css:19]: sure

File: src/frontend/components/postsContent/PostsContent.css (hunk 1 above)
Comment (SRUJANA-PENUGONDA13) [reply src/frontend/components/postsContent/PostsContent.css]: sure

**Pull request merged**